src/Toolbox.php

```diff
@@ -1371,6 +1373,57 @@ public static function callCurl(
         /** @var array $CFG_GLPI */
         global $CFG_GLPI;
 
+        try {
+            return self::doCallCurl($url, $eopts, $msgerr, $curl_error, $check_url_safeness, $curl_info);
+        } catch (CurlException $e) {
+            ErrorHandler::logCaughtException($e);
```

Comment:
In the future, I would like to remove usages of `ErrorHandler::logCaughtException()` that requires to maintain static usage of the error handler.

```suggestion
            /** @var LoggerInterface $PHPLOGGER */
            global $PHPLOGGER;
            $PHPLOGGER->error(
                $e->getMessage(),
                ['exception' => $e]
            );
```